.github/workflows/auto-merge.yml

```diff
@@ -1,4 +1,4 @@
-name: Explore dependabot metadata
+name: Auto-merge Dependabot Python updates
 
 on: pull_request_target
```

Comment:
question: What's the need for `pull_request_target` over `pull_request`? The [example in GitHub's docs](https://docs.github.com/en/code-security/supply-chain-security/keeping-your-dependencies-updated-automatically/automating-dependabot-with-github-actions#enable-auto-merge-on-a-pull-request) uses `pull_request`.

Reply:
Just a mistake left in from testing, fixed and rebased.